### ISSUE TITLE
Embed info about IRC (?)

### ISSUE THREAD

**Issue by tripu** [w3c/Unitas#34]
Channels used by the group?

**Comment by chaals** [comment]
Yes please. General communication channels is a really valuable set of information. Some groups have one of several things, others like Web Platform have multiple mailing lists with a purpose for each, and I believe at least WAI still regularly uses different lists with different posting/visibility properties, even beyond "public list and member list".

**Comment by tripu** [comment]
(Title and description updated to mention &ldquo;communication channels&rdquo; in general, not just IRC.)
